kiva/agg/src/graphics_context.i

```diff
@@ -848,8 +850,7 @@ namespace kiva {
                 else:
                     bmp = self.bmp_array
 
-                from kiva import compat
-                img = compat.pilfromstring(pilformat, size, bmp.tostring())
+                img = Image.frombytes(pilformat, size, bmp.tobytes())
```

Comment:
`bmp` is a numpy array. `tobytes` is the new, more correct, name for `tostring` (which was kept for compatibility with old code like this but is now deprecated in recent versions of numpy)